From 4ba61c59d3488c263d106d486b656854a57ad79f Mon Sep 17 00:00:00 2001
From: Jens Rehsack <sno@netbsd.org>
Date: Thu, 13 Aug 2020 15:26:30 +0200
Subject: [PATCH 1/2] conf/Unix.mk: remove fixed command definitions

For cross compiling in Yocto or with appropriate SDKs, commands like
`$CC` are reasonably predefined.

Upstream-Status: Inappropriate

Signed-off-by: Jens Rehsack <sno@netbsd.org>
---
 conf/Unix.mk | 10 +++-------
 1 file changed, 3 insertions(+), 7 deletions(-)

diff --git a/conf/Unix.mk b/conf/Unix.mk
index 02f2b2b..05979fc 100644
--- a/conf/Unix.mk
+++ b/conf/Unix.mk
@@ -37,23 +37,19 @@ RM = rm -f
 MKDIR = mkdir -p
 
 # C compiler and flags.
-CC = cc
-CFLAGS = -W -Wall -Os -fPIC
 CCOUT = -c -o 
 
 # Static library building tool.
-AR = ar
 ARFLAGS = -rcs
 AROUT =
 
 # DLL building tool.
-LDDLL = cc
+LDDLL = $(CCLD)
 LDDLLFLAGS = -shared
 LDDLLOUT = -o 
 
 # Static linker.
-LD = cc
-LDFLAGS = 
+LD = $(CCLD)
 LDOUT = -o 
 
 # C# compiler; we assume usage of Mono.
@@ -63,7 +59,7 @@ RUNT0COMP = mono T0Comp.exe
 # Set the values to 'no' to disable building of the corresponding element
 # by default. Building can still be invoked with an explicit target call
 # (e.g. 'make dll' to force build the DLL).
-#STATICLIB = no
+STATICLIB = no
 #DLL = no
 #TOOLS = no
 #TESTS = no
-- 
2.17.1

